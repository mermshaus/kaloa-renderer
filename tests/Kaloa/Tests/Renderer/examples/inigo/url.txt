[url=https://www.example.org/]Link[/url]

[url="https://www.example.org/"]Link[/url]

[url href="https://www.example.org/"]Link[/url]

[url href="https://www.example.org/" title="Foo"]Link[/url]

[url="https://www.example.org/?x&y" title="Foo>bar"]Link[/url]



[link=https://www.example.org/]Link[/link]

[link="https://www.example.org/"]Link[/link]

[link href="https://www.example.org/"]Link[/link]

[link href="https://www.example.org/" title="Foo"]Link[/link]

[link="https://www.example.org/?x&y" title="Foo>bar"]Link[/link]
