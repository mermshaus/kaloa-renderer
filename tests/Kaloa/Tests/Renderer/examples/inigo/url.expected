<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Foo">Link</a></p>

<p><a href="https://www.example.org/?x&amp;y" title="Foo&gt;bar">Link</a></p>

<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Open &quot;https://www.example.org/&quot;">Link</a></p>

<p><a href="https://www.example.org/" title="Foo">Link</a></p>

<p><a href="https://www.example.org/?x&amp;y" title="Foo&gt;bar">Link</a></p>
